tests/test_ops.py

```diff
@@ -76,6 +76,19 @@ def test_elemwise_ops_1_arg(arr3d, func_name):
     assert_allclose(actual.todense(), expected)
 
 
+@pytest.mark.parametrize(
+    "func_name", ["real", "imag", "conj"]
+)
+def test_elemwise_complex_ops_1_arg(func_name):
```

Comment:
Could we test more dtypes? I don't know if Finch would do the right thing for those.